src/main/java/org/amahi/anywhere/activity/ServerFileVideoActivity.java

```diff
@@ -24,59 +24,75 @@
 import android.content.Context;
 import android.content.Intent;
 import android.content.ServiceConnection;
+import android.content.res.Configuration;
 import android.graphics.PixelFormat;
+import android.os.Build;
 import android.os.Bundle;
 import android.os.Handler;
 import android.os.IBinder;
+import android.util.Log;
 import android.view.MenuItem;
 import android.view.SurfaceHolder;
 import android.view.SurfaceView;
 import android.view.View;
 import android.view.ViewGroup;
+import android.widget.FrameLayout;
 import android.widget.MediaController;
 
-import com.squareup.otto.Subscribe;
-
 import org.amahi.anywhere.AmahiApplication;
 import org.amahi.anywhere.R;
-import org.amahi.anywhere.bus.BusProvider;
-import org.amahi.anywhere.bus.VideoFinishedEvent;
-import org.amahi.anywhere.bus.VideoSizeChangedEvent;
-import org.amahi.anywhere.bus.VideoStartedEvent;
-import org.amahi.anywhere.server.client.ServerClient;
 import org.amahi.anywhere.server.model.ServerFile;
 import org.amahi.anywhere.server.model.ServerShare;
 import org.amahi.anywhere.service.VideoService;
-import org.amahi.anywhere.util.Android;
 import org.amahi.anywhere.util.Intents;
 import org.amahi.anywhere.util.ViewDirector;
 import org.amahi.anywhere.view.MediaControls;
+import org.videolan.libvlc.IVLCVout;
+import org.videolan.libvlc.Media;
+import org.videolan.libvlc.MediaPlayer;
 
-import java.util.Arrays;
-import java.util.HashSet;
-import java.util.Set;
 import java.util.concurrent.TimeUnit;
 
-import javax.inject.Inject;
-
 /**
  * Video activity. Shows videos, supports basic operations such as pausing, resuming, scrolling.
  * The playback itself is done via {@link org.amahi.anywhere.service.VideoService}.
  * Backed up by {@link android.view.SurfaceView} and {@link org.videolan.libvlc.LibVLC}.
  */
-public class ServerFileVideoActivity extends Activity implements ServiceConnection,
-	SurfaceHolder.Callback,
+public class ServerFileVideoActivity extends Activity implements
+	ServiceConnection,
 	Runnable,
 	MediaController.MediaPlayerControl,
-	View.OnSystemUiVisibilityChangeListener
-{
-	@Inject
-	ServerClient serverClient;
```

Comment:
Just for reference: here ^ is the instance injection that was removed. @megabitdragon